scss/_regions/_modal.scss

```diff
@@ -62,12 +62,16 @@
   position: absolute;
   top: 0;
   right: $base-spacing;
-  font-size: 45px;
+  font-size: 36px;
```

Comment:
Should these font sizes correlate with the new font size variables you created with Neue 6?